code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -69,6 +69,38 @@ Contains most of the procs that are called when a mob is attacked by something
 
 	return siemens_coefficient
 
+/**
+	Helper proc to determine if a mob is fire resistant (wont catch on fire)
+
+	For fire protection, suits provide full body protection with > 100 fire protection
+	without a suit, you need >100 fire protection from head, chest, gloves and shoes seperately (otherwise the fire can get in)
+*/
+/mob/living/carbon/human/proc/update_fire_resistant(equipped)
```

Comment:
Give the armors fire hard armor and then make a check on the human for it to remove the need for this proc.